README.md

```diff
@@ -94,15 +48,19 @@ If you have security issues to report, please refer to our [Responsible Disclosu
 ### :wrench: Tools
 
 - [CLI](https://github.com/sequelize/cli)
-- [With TypeScript](https://sequelize.org/master/manual/typescript.html)
+- [With TypeScript](https://sequelize.org/docs/v7/other-topics/typescript)
```

Comment:
Do we want this to be the v7 version? As this is the README that people will also see when they are looking for the stable release

Reply:
The new website has a warning to redirect users to v6, I think it's fine :) https://new.sequelizejs.com/docs/v7/other-topics/typescript/

Alternatively we could remove it from here, the new documentation is going to make that page mostly redundant because most examples will include a TS version

Reply:
Ah yes, I forgot about that. Then it should be fine to keep it like this